File: 프로그래머스/lv0/120895. 인덱스 바꾸기/인덱스 바꾸기.java
class Solution {
    public String solution(String my_string, int num1, int num2) {
        // 1. 기존 스트링을 배열로 변환
        String[] arr = my_string.split("");

        // 2. 새로운 배열 만들어서 기존 배열 복사
        String[] answer = my_string.split("");

        // 3. 바꾸고 싶은 문자열 새로운 배열에 넣기
        answer[num1] = arr[num2];
        answer[num2] = arr[num1];

        return String.join("", answer);
    }
}

File: 프로그래머스/unrated/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    answer = ''
    
    next = ''
    for i in range(len(my_string)):
        if i == num1:        
            next = my_string[i]
            answer += my_string[num2]
            continue
        elif i == num2:
            answer += next
            continue
        answer += my_string[i]
            
    return answer